ktor-client/ktor-client-java/jvm/src/io/ktor/client/engine/java/JavaHttpWebSocket.kt

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.engine.java
+
+import io.ktor.client.engine.*
+import io.ktor.client.features.*
+import io.ktor.client.features.websocket.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.http.cio.websocket.*
+import io.ktor.util.*
+import io.ktor.util.date.*
+import io.ktor.utils.io.core.*
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.*
+import kotlinx.coroutines.future.*
+import java.net.http.*
+import java.nio.*
+import java.time.*
+import java.util.*
+import java.util.concurrent.*
+import kotlin.coroutines.*
+import kotlin.text.String
+import kotlin.text.toByteArray
+
+private val ILLEGAL_HEADERS = TreeSet(String.CASE_INSENSITIVE_ORDER).apply {
+    addAll(
+        setOf(
+            "Sec-WebSocket-Accept",
```

Comment:
Don't we have them in `HttpHeaders`?

Reply:
Now all of these constants are replaced by corresponding `HttpHeaders`.